src/app/series/directives/series-podcast.component.ts

```diff
@@ -101,4 +101,21 @@ export class SeriesPodcastComponent implements OnDestroy, DoCheck {
     }
   }
 
+  resetNewFeedUrlOnCancel(confirm) {
+    if (!confirm) {
+      this.podcast.set('newFeedUrl', this.podcast.original['newFeedUrl']);
+    }
+  }
+
+  get newFeedUrlConfirm() {
+    if (this.podcast) {
+      return {
+        confirm: AdvancedConfirmText.newFeedUrl(
+          !this.podcast.isNew && !this.podcast.invalid('newFeedUrl') && this.podcast.changed('newFeedUrl'),
```

Comment:
I have a feeling that checking for `invalid()` may not work out, in the future.  Assuming we allow saving invalid stuff to CMS.  But since this is a series and not a story ... I haven't quite hashed that out in my brain yet.